docs/contributing/doc_guide.rst

```diff
@@ -1112,24 +1117,48 @@ Narrative documentation guidelines
 Troubleshooting
 ===============
 
-* Errors like ``WARNING: py:class reference target not found: ...``
-  occur when Sphinx is unable to interpret text as a Python object.
-
-  - If the text is meant to be a code sample, make sure that the text is
-    in double back ticks.
-
-  - If the text should refer to a code object, make sure that the full
-    namespace is provided correctly.
-
-  - If the text is in the type line for a parameter in a docstring, then
-    either:
-
-    - Change the type line to include only Python types and words or
-      patterns defined in ``nitpick_ignore_regex`` in |docs/conf.py|_
-      (preferred), or
-
-    - Add the text as a regular expression to ``nitpick_ignore_regex``
-      so that Sphinx knows not to issue a warning for this text.
+This section contains suggestions about how to fix common documentation
+errors and warnings.
+
+Reference target not found
+--------------------------
+
+Warnings like ``py:obj reference target not found`` occur when Sphinx_
+attempts to interpret text as a Python object, but is unable to do so.
+For example, if a docstring includes ```y```, Sphinx will attempt
+to link to an object named ``y``. If there is no object named ``y``,
+then Sphinx will issue this warning, which gets treated like an error.
+
+If the text is meant to be an in-line code snippet, surround it with
+double backticks instead of single backticks.
+
+This warning may also occur when there is an extra space between a
+|role| and the argument it is intended to act on. For example, this
+warning would be fixed by changing ``:math: `y``` to
+``:math:`y```.
+
+When the text is meant to represent a code object like
+```astropy.units.Quantity```, then this warning usually indicates
+either a typo or an error in the namespace. For example, the warning
+resulting from ```astropy.Quantity``` can be resolved by changing it to
+```astropy.units.Quantity```.
+
+This warning sometimes occurs in the type line of a |parameter| in a
+docstring. Sphinx attempts to link words in type lines to code objects.
+Type lines are intended to provide concise information about allowed
+types, sizes, shapes, physical types, and default values of a parameter.
+To resolve this warning, first move information about the *meaning* of
+a parameter from the type line into the parameter description that
+begins on the following line. To expand the list of allowed words or
+patterns in type lines, add a regular expression to
+``nitpick_ignore_regex`` in |docs/conf.py|_.
+
```

Comment:
Add a paragraph on how the `reference target not found` error might be caused by a missing `api_static` page?